<!DOCTYPE html>
<html>
    <head>
        <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0"> -->
        <!-- use above code can fit the size of the screen -->
        <meta name="viewport">
        <meta http-equiv='cache-control' content='no-cache'> 
        <meta http-equiv='expires' content='0'> 
        <meta http-equiv='pragma' content='no-cache'>
        <title>I-Shu Wang</title>
        <link rel="stylesheet" type="text/css" href="style.css">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
        
    </head>
<body>
    <div class="wapper">
        <div class="topnav">
            <a href="#about">About</a>
            <a href="#education">Education</a>
            <a href="#experience">Experience</a>
            <a href="#projects">Projects</a>
            <a href="#skills">Skills</a>
            <a href="#Publication">Publication</a>
            <a href="#contact">Contact</a>
            <a class=resume href="I_Shu_Wang_Resume.pdf" download="I_Shu_Wang">Resume</a>
        </div>
        <div class="profile" id="about">
            <div class="imgBx">
                <img src="image/IMG_3490.JPG">
            </div>
            <div class="text">
                <h1>Hi, this is I-Shu Wang</h1>              
                <h2>
                    <span>
                        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;A motivated graduate student major in Quantitative Biology and Bioinformatics at Carnegie Mellon University, with an expected graduation in Dec 2023. 
                        Complemented by a Master's degree in Biochemical Science and Technology from National Taiwan University. 
                        Offering a strong academic background and hands-on experience in genomic research, data analysis, and bioinformatics. 
                        Proficient in utilizing various computational tools and programming languages, including shell script, Python, Go, R, to derive meaningful insights from biological data. 
                        Seeking a full-time position to apply my skills and contribute to cutting-edge research in the field of bioinformatics.
                    </span>
                </h2>
            </div>   
        </div>
        <!-- Education timeline -->
        <div class="timeline" id="education">
            <div class="timeline_title">
                <h2>Education</h2>
            </div>
            <!-- CMU -->
            <div class="timeline__component">
                <div class="timeline__date timeline__date--right">Dec 2023</div>
            </div>
            <div class="timeline__middle">
                <div class="timeline__point"></div>
            </div>
            <div class="timeline__component timeline__component--bg">
                <h2 class="timeline__title">Carnegie Mellon University</h2>
                <p class="timeline__paragraph">
                    Pittsburgh, PA <br>
                    Master of Science in Quantitative Biology And Bioinformatics
                    <ul class="timeline__list_first_item">
                        <li>Current GPA: 4.01/4.33</li>
                        <li>Relative Courses: (* Fall Semester)</li> 
                        <ul class="timeline__list_second_item">  
                            <li>*Computational Molecular Biology and Genomics</li>
                            <li>Bioinformatics Data Integration Practicum</li>
                            <li>Data Analysis for Biological Sciences</li>
                            <li>Algorithms & Advanced Data Structures</li>
                            <li>Fundamentals of Bioinformatics</li>
                            <li>*Introduction to Deep Learning</li>
                            <li>Programming for Scientists</li>  
                            <li>*Computational Medicine</li>
                            <li>Machine Learning</li>
                        </ul>
                    </ul>
                </p>
            </div>
            <!-- NTU -->
            <div class="timeline__component timeline__component--bg">
                <h2 class="timeline__title">National Taiwan University</h2>
                <p class="timeline__paragraph">
                    Taipei, Taiwan <br>
                    Master of Science in Biochemical Science & Technology
                    <ul class="timeline__list_first_item">
                        <li>Member of Food Chemistry Lab, supervised by Prof. Nan-Wei Su</li>
                        <li>Thesis: The effect of <i>qdoI</i> on the flavon-3-ol phosphorylation by <i>Bacillus subtilis</i> BCRC80517</li>
                        <li>Earned Excellence Award from 2020 Agricultural Chemical Society of Taiwan Annual Poster Competition Sessions</li>
                    </ul>
                </p>
                </p>
            </div>
            <div class="timeline__middle">
                <div class="timeline__point"></div>
            </div>
            <div class="timeline__component">
                <div class="timeline__date">Jun 2020</div>
            </div>
            <!-- CCU -->
            <div class="timeline__component">
                <div class="timeline__date timeline__date--right">June 2018</div>
            </div>
            <div class="timeline__middle">
                <div class="timeline__point"></div>
            </div>
            <div class="timeline__component timeline__component--bg">
                <h2 class="timeline__title">National Chung Cheng University </h2>
                <p class="timeline__paragraph">
                Chiayi, Taiwan <br>
                Bachelor of Science in Biomedical Science
                    <ul class="timeline__list_first_item">
                        <li>Certificate in Functional Genomics Program</li>
                        <li>Relative Courses:</li>
                            <ul class="timeline__list_second_item">
                                <li>Introduction to Systems Biology</li>
                                <li>Introduction to Bioinformatics</li>
                                <li>Introduction to Biostatistics</li>
                                <li>Molecular Biology</li>
                                <li>Microbiology</li>
                                <li>Cell Biology</li>
                                <li>Biochemistry</li>
                                <li>Immunology</li>
                                <li>Virology</li>
                                <li>Genetics</li>
                            </ul>
                    </ul>
                </p>
            </div>
        </div>

        <!-- Experience timeline -->
        <div class="timeline" id="experience">
            <div class="timeline_title">
                <h2>Experience</h2>
            </div>
            <!-- CMU Graduate Researcher -->
            <div class="timeline__component">
                <div class="timeline__date timeline__date--right">Jan 2023 - Aug 2023</div>
            </div>
            <div class="timeline__middle">
                <div class="timeline__point"></div>
            </div>
            <div class="timeline__component timeline__component--bg">
                <h2 class="timeline__title">Graduate Researcher</h2>
                <p class="timeline__paragraph">
                    Carnegie Mellon University
                    <ul class="timeline__list_first_item">
                        <li>
                            Analyzing PacBio and NanoPore RNA sequence data to determine expression levels of various isoforms in Linux
                        </li>
                        <li>
                            Built a pipeline using Snakemake and parallel programming for analyzing alternative splicing on RNA-seq data
                        </li>
                        <li>
                            Conduct comprehensive data analysis to draw conclusive insights on expression levels from sequencing data
                        </li>
                        <li>
                            Developed an interactive web application to display the isoform categories for each gene
                        </li>
                    </ul>
                </p>
            </div>
            <!-- Research Assistant -->
            <div class="timeline__component timeline__component--bg">
                <h2 class="timeline__title">Research Assistant </h2>
                <p class="timeline__paragraph">
                    NGS High Throughput Genomics Core, Academia Sinica
                    <ul class="timeline__list_first_item">
                        <li>
                            Operated and maintained Illumina sequencers to get high quality sequencing data
                        </li>
                        <li>
                            Demultiplexed sequencing data and used Fastqc to examine quality of sequencing data
                        </li>
                        <li>
                            Proficiently prepared Next Generation Sequencing libraries employing diverse assays for both DNA and RNA samples
                        </li>
                    </ul>
                </p>
            </div>
            <div class="timeline__middle">
                <div class="timeline__point"></div>
            </div>
            <div class="timeline__component">
                <div class="timeline__date">Feb 2021 - Aug 2021</div>
            </div>
            <!-- Graduate Researcher in NTU -->
            <div class="timeline__component">
                <div class="timeline__date timeline__date--right">Aug 2018 - Jun 2020</div>
            </div>
            <div class="timeline__middle">
                <div class="timeline__point"></div>
            </div>
            <div class="timeline__component timeline__component--bg">
                <h2 class="timeline__title">Graduate Researcher </h2>
                <p class="timeline__paragraph">
                    National Taiwan University
                    <ul class="timeline__list_first_item">
                        <li>
                            Executed CRISPR-Cas9 on an enzyme gene in Bacillus subtilis, preventing the degradation of the target molecules
                        </li>
                        <li>
                            Researched Basic Local Alignment Search Tool (BLAST) and NCBI to build plasmid for gene knockout system
                        </li>
                        <li>
                            Analyzed sanger sequencing data by Vector NTI to check success of gene editing
                        </li>  
                    </ul>
                </p>
            </div>
            <!-- Teaching Assistant -->
            <div class="timeline__component timeline__component--bg">
                <h2 class="timeline__title">Teaching Assistant </h2>
                <p class="timeline__paragraph">
                    National Taiwan University
                    <ul class="timeline__list_first_item">
                        <li>
                            Collaborated with a team of 4 to teach and train over 30 students to learn experiments
                        </li>
                        <li>
                            Designed project outline and moderated discussion for students’ final projects
                        </li>
                    </ul>
                </p>
            </div>
            <div class="timeline__middle">
                <div class="timeline__point"></div>
            </div>
            <div class="timeline__component">
                <div class="timeline__date">Aug 2018 - Jan 2019</div>
            </div>

            <!-- Intern -->
            <div class="timeline__component">
                <div class="timeline__date timeline__date--right">July 2015 - Aug 2015</div>
            </div>
            <div class="timeline__middle">
                <div class="timeline__point"></div>
            </div>
            <div class="timeline__component timeline__component--bg">
                <h2 class="timeline__title">Summer Intern for Research Assistant</h2>
                <p class="timeline__paragraph">
                    Academia Sinica
                    <ul class="timeline__list_first_item">
                        <li>
                            Researched on Oncology to treated 2-deoxy-glucose to cancer cells, resulting in the level of heat shock protein and mitosis cycle changed
                        </li>
                        <li>
                            Tested protein level through western blotting, and processed immunofluorescence to visualize mitosis status
                        </li>
                    </ul>
                </p>
            </div>
        </div>
        <!-- Projects -->
        <div class="cards" id="projects">
            <div class="cards_title">
                <h2>Projects</h2>
            </div>
            <!-- 1 -->
            <div class="card">
                <div class="card-inner">
                    <div class="card-front">
                        <p class="flip-card-font-title">
                            A python script for bacterial genome analysis
                        </p>
                        <p class="flip-card-font-content">
                            Carnegie Mellon University<br>
                            Pittsburgh, PA<br>
                            Oct 2022
                        </p>
                    </div>
                    <div class="card-back">
                            <ul class="flip-card-back-content">
                                <li>
                                    Created a Python script that perform a series of processes from examining whole genomics for bacteria, 
                                    translating DNA sequences to protein sequences, to using BLAST to compare the protein sequences
                                </li>
                                <li>
                                    Scanned the sequence data to get the high potential Open Reading Frame sequences which length were longer 
                                    than 600 pb and utilized Biopython to establish connections with BLAST and NCBI databases.
                                </li>
                                <li>
                                    <a href="https://github.com/wang9685/PythonScriptForWholeGenome">GitHub</a>
                                </li>
                                <li>
                                    Skill:
                                    <ul class="flip-card-back-skill">
                                        <li>Python</li>
                                        <li>BioPython</li>
                                        <li>BLAST</li>
                                    </ul>
                                </li>
                            </ul>
                    </div>
                </div>
            </div>
            <!-- 2 -->
            <div class="card">
                <div class="card-inner">
                    <div class="card-front">
                        <p class="flip-card-font-title">
                            Data analysis and visualization on oncology data
                        </p>
                        <p class="flip-card-font-content">
                            Carnegie Mellon University<br>
                            Pittsburgh, PA<br>
                            Dec 2022
                        </p>
                    </div>
                    <div class="card-back">
                        <ul class="flip-card-back-content">
                            <li>
                                Conducted comprehensive statistical analysis on oncology data using Python, h
                                arnessing the power of essential libraries such as SciPy, Pandas, NumPy, Seaborn, Statsmodels, Plotly, and Matplotlib
                            </li>
                            <li>
                                Employed advanced statistical techniques and data manipulation to derive actionable insights 
                                and trends, which were then effectively communicated through data visualizations
                            </li>
                            <li>
                                Skill:
                                <ul class="flip-card-back-skill">
                                    <li>Python</li>
                                    <li>Matplotlib</li>
                                    <li>Statsmodels</li>
                                    <li>SciPy</li>
                                    <li>Seaborn</li>
                                    <li>plotly</li>
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
            <!-- 3 -->
            <div class="card">
                <div class="card-inner">
                    <div class="card-front">
                        <p class="flip-card-font-title">
                            Simulation of intercellular viral infection 
                            via cell-to-cell transmission 
                        </p>
                        <p class="flip-card-font-content">
                            Carnegie Mellon University<br>
                            Pittsburgh, PA<br>
                            Dec 2022
                        </p>
                    </div>
                    <div class="card-back">
                        <ul class="flip-card-back-content">
                            <li>
                                Contributed to the development of a dynamic simulation model in the Go programming language, 
                                enabling users to input critical parameters, including those related to drug or treatment strategies, derived from biological data
                            </li>
                            <li>
                                This user-centric model offered a powerful tool for individuals and researchers to assess the impact and efficacy of specific treatments in 
                                combating viral infections within host cells, with the assurance of rigorous unit testing for model accuracy and functionality.
                            </li>
                            <li>
                                Worked with team of 4
                            </li>
                            <li>
                                Skill:
                                <ul class="flip-card-back-skill">
                                    <li>GO Language</li>
                                    <li>Unit tests</li>
                                    <li>Parallel Programming</li>
                                    <li>Git</li>
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>

            <!-- 4 -->
            <div class="card">
                <div class="card-inner">
                    <div class="card-front">
                        <p class="flip-card-font-title">
                            Construction of a Transcriptome Assembly and Annotation Pipeline
                        </p>
                        <p class="flip-card-font-content">
                            Carnegie Mellon University<br>
                            Pittsburgh, PA<br>
                            Apr 2023
                        </p>
                    </div>
                    <div class="card-back">
                        <ul class="flip-card-back-content">
                            <li>
                                Designed and implemented AnnoGenie, a genome annotation pipeline that harnessed evolutionary similarities between species to annotate the gray wolf genome. This achievement was 
                                realized by building and deploying the pipeline on Bridge2, a cloud-based high-performance computer, running under Linux
                            </li>
                            <li>
                                The pipeline's core components were scripted in shell scripting, involved utilizing tools such as FastQC, Trimmomatic, HISAT2, Samtools, StringTie, gffRead, 
                                and BLAST to create an innovative annotation approach. The project successfully bridged the gap 
                                for species lacking transcript assemblies and RNA-seq data, thereby facilitating efficient genome annotation in novel species. 
                            </li>
                            <li>
                                Worked with team of 4.
                            </li>
                            <li>
                                Skill:
                                <ul class="flip-card-back-skill">
                                    <li>High Performance Computing</li>
                                    <li>Pipeline Construction</li>
                                    <li>Shell Language</li>
                                    <li>Genome Alignment</li>
                                    <li>Genome Annotation</li>
            
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
            <!-- 5 -->
            <div class="card">
                <div class="card-inner">
                    <div class="card-front">
                        <p class="flip-card-font-title">
                            Analysis of differential gene expression and pathway enrichment in multiple cancers
                        </p>
                        <p class="flip-card-font-content">
                            Carnegie Mellon University<br>
                            Pittsburgh, PA<br>
                            Apr 2023
                        </p>
                    </div>
                    <div class="card-back">
                        <ul class="flip-card-back-content">
                            <li>
                                Utilized RNA-seq data from The Cancer Genome Atlas (TCGA) to analyze gene expression in 12 prevalent cancer types. 
                                Conducted differential gene expression analysis with DESeq2, revealing shared gene signatures across cancers. 
                                Employed Enrichr for pathway-based gene clustering and applied Fisher's method for further statistical analysis. 
                            </li>
                            <li>
                                The project yielded insights into common molecular pathways involved in cancer development and progression, with potential implications 
                                for targeted therapies and personalized treatments, ultimately contributing to enhanced patient outcomes in oncology
                            </li>
                            <li>
                                Worked with team of 4.
                            </li>
                            <li>
                                <a href=https://github.com/luotao9728/pathway_enrichment>GitHub</a>
                            <li>
                                Skill:
                                <ul class="flip-card-back-skill">
                                    <li>TCGA database</li>
                                    <li>DESeq2</li>
                                    <li>Enrichr</li>
                                    <li>Python</li>
                                    <li>R</li>
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
            <!--6-->
            <div class="card">
                <div class="card-inner">
                    <div class="card-front">
                        <p class="flip-card-font-title">
                            Frame-Level Speech Recognition 
                        </p>
                        <p class="flip-card-font-content">
                            Carnegie Mellon University<br>
                            Pittsburgh, PA<br>
                            Sep 2023
                        </p>
                    </div>
                    <div class="card-back">
                        <ul class="flip-card-back-content">
                            <li>
                                Designed and implemented a multilayer perceptron (MLP) for frame-level speech recognition, predicting frame-level phonetic transcriptions from raw Mel 
                                Frequency Cepstral Coefficients (MFCCs). The MLP, developed using PyTorch, learned feature 
                                representations and nonlinear classification boundaries, with a focus on discriminating between phoneme class labels. 
                            </li>
                            <li>
                                Operating within a limited parameter budget of 25MB, extensive optimization efforts were conducted, including testing different loss functions, 
                                architectures, and learning rates, aimed at achieving a high accuracy threshold of 90%. 
                                The entire model development and training process were processed on the Google Cloud Platform.
                            </li>
                            
                                Skill:
                                <ul class="flip-card-back-skill">
                                    <li>Pytorch</li>
                                    <li>Google Cloud Platform</li>
                                    <li>Cloud Computing</li>
                                    <li>Deep Learning</li>
                                    
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
            <!--7-->
            <div class="card">
                <div class="card-inner">
                    <div class="card-front">
                        <p class="flip-card-font-title">
                            Face Verification and recognition  
                        </p>
                        <p class="flip-card-font-content">
                            Carnegie Mellon University<br>
                            Pittsburgh, PA<br>
                            Oct 2023
                        </p>
                    </div>
                    <div class="card-back">
                        <ul class="flip-card-back-content">
                            <li>
                                Designed and implemented a multilayer perceptron (MLP) for frame-level speech recognition, predicting frame-level phonetic transcriptions from raw Mel 
                                Frequency Cepstral Coefficients (MFCCs). The MLP, developed using PyTorch, learned feature 
                                representations and nonlinear classification boundaries, with a focus on discriminating between phoneme class labels. 
                            </li>
                            <li>
                                Operating within a limited parameter budget of 25MB, extensive optimization efforts were conducted, including testing different loss functions, 
                                architectures, and learning rates, aimed at achieving a high accuracy threshold of 90%. 
                                The entire model development and training process were processed on the Google Cloud Platform.
                            </li>
                            
                                Skill:
                                <ul class="flip-card-back-skill">
                                    <li>Pytorch</li>
                                    <li>Google Cloud Platform</li>
                                    <li>Cloud Computing</li>
                                    <li>Deep Learning</li>
                                    
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>





        </div>
        <div class="Set_list" id="skills">
            <div class="list_title">
                <h2>Skills</h2>
            </div>
            <div class="content_title">Programming Language</div>
            <div class="content">
                <p>Python, R, GO Language, SQL, Shell script</p>
            </div>
            <div class="content_title">Computational Skills</div>
            <!-- <button type="button" onclick="clickandexpand()" class="collapsible">Computational Skills</button> -->
            <div class="content">
                <p>Experience in Git, paralle programming, linux enviornment, AWS, Google Cloud Platform</p>
            </div>
            <div class="content_title">Bioinformatics Tools</div>
            <!-- <button type="button" onclick="clickandexpand()" class="collapsible">Bioinformatics Tools</button> -->
            <div class="content">
                <p>NCBI, BLAST, TCGA, Snakemake, Bioconductor, BioPython, DESeq2, Enrichr, various alignment and annotation tools</p>
            </div>
            <div class="content_title">Data Analysis Packages</div>
            <!-- <button type="button" onclick="clickandexpand()" class="collapsible">Data Analysis Packages</button> -->
            <div class="content">
                <p>Numpy, Pandas, SciPy, Seaborn, Statsmodels, plotly, ggplot, Matplotlib, Scikit-Learn, Pytorch, dash, RShiny</p>
            </div>
        </div>
        <!-- Publication -->
        <div class="Set_list" id="Publication">
            <div class="list_title">
                <h2>Publication</h2>
            </div>
            <div class="papers">
                <ul class="list_parper">
                    <li>
                        Wang, C.W., Tsai, H.Y., Hsu, C., Hsieh, C.C., <b>Wang, I.S.</b>, Chang, C.F., & Su, N.W. (2024). Structure-specific metabolism of flavonol molecules by Bacillus subtilis var. Natto BCRC 80517. Food Chemistry, 136975–136975. https://doi.org/10.1016/j.foodchem.2023.136975
                    </li>
                </ul>
            </div>
        </div>
        <!-- Contact me -->
        <div class="Contact" id="contact">
            <div class="contact_title">
                <h2>Contact Me</h2>
            </div>
            <div class="icons">
                <a href="https://www.linkedin.com/in/i-shu-wang-085a201b9/"><i class="fa fa-linkedin" style="font-size:24px"></i></a>
            </div>
            <div class="icons">
                <a href="mailto:ishuwang9685@gmail.com"><i class="fa fa-envelope" style="font-size:24px"></i></a>
            </div>

        </div>
    </div>
    
    <footer>
    </footer>
    

    <script type="text/javascript" src="script.js"></script>
</body>
</html>

